lib/suspenders.rb

```diff
@@ -1,5 +1,6 @@
 require 'suspenders/version'
 require 'suspenders/generators/app_generator'
+require 'suspenders/generators/static_generator'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.